### PR TITLE
Adding start and end times to the alerting flow

### PR DESCRIPTION
**Pull request by smcmurtry**
# Summary | Résumé

What I did:
- added "From" and "To" input boxes that allow the start and end hour for an alert to be selected
- these get used to save `alert_datetime_start_x` and `alert_datetime_end_x` to the session
- hours are formatted differently for french and english
- added the start and end times to the "confirm" and "confirmation" pages
- these start and end times are posted from the "confirm" page, so hopefully they are making it into the db correctly
- bottom border below each group of form fields

Still to do:
- update the `select` element to use markup and styling from gov.uk
- form validation 

Screen Shot 2021-03-12 at 3 59 28 PM

### CODE REVIEW

File: outbreaks/forms.py [hunk 1]
```diff
@@ -27,22 +33,40 @@ def __init__(self, num_dates=1, *args, **kwargs):
...
                 min_value=1,
-                max_value=datetime.now().month,
+                max_value=12,
                 widget=forms.TextInput,
             )
             self.fields[f"year_{i}"] = forms.IntegerField(
                 label=_("Year"),
                 min_value=2021,
-                max_value=2021,
+                max_value=datetime.now().year,
```
Comment (andrej-vasilj) [inline outbreaks/forms.py:42]: nice change! makes sense :)

**Review by andrej-vasilj — APPROVED** [review]
Looks solid to me! It'll will need some more validations/error states for the new edge cases, but aside from that it looks like it checks all the boxes.

**Comment by dsamojlenko** [comment]
Holding back for now... redesign of this screen underway.

**Comment by dsamojlenko** [comment]
closing as the entire feature has been redesigned.

**Pull request closed without merging**